apps/snippets/models.py

```diff
@@ -1,47 +1,101 @@
 from django.db import models
 from modelcluster.fields import ParentalKey
 from modelcluster.models import ClusterableModel
-from wagtail.admin.edit_handlers import (FieldPanel, InlinePanel,
-                                                PageChooserPanel)
+from wagtail.admin.edit_handlers import FieldPanel
+from wagtail.admin.edit_handlers import InlinePanel
+from wagtail.admin.edit_handlers import PageChooserPanel
+from wagtail.admin.edit_handlers import StreamFieldPanel
+from wagtail.core.fields import StreamField
 from wagtail.core.models import Orderable
 from wagtail.snippets.models import register_snippet
 
+from apps.contrib.translations import TranslatedField
+from apps.snippets import blocks as snippets_blocks
+
 
 class MenuItem(models.Model):
+
+    title_de = models.CharField(
+        max_length=255,
+    )
+    title_en = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+    )
+
     link_page = models.ForeignKey(
         'wagtailcore.Page',
         related_name='+',
+        blank=True,
+        null=True,
+        help_text=(
+            'Creates a link to a single wagtail page. '
+            'Leave empty if you add subpages. '
+            'In the footer items with subpages will not be shown.'
+        ),
         on_delete=models.CASCADE
     )
-    title = models.CharField(
-        max_length=255, verbose_name="Title")
+
+    subpages = StreamField(
+        [
+            ('link', snippets_blocks.LinkBlock())
+        ],
+        blank=True,
+        null=True,
+        help_text=(
+            'These Links will be displayed in a dropdown menu'
+        ),
+        verbose_name='Submenu'
+    )
+
+    title = TranslatedField(
+        'title_de',
+        'title_en'
+    )
+
+    class Meta:
+        abstract = True
 
     @property
     def url(self):
-        return self.link_page.url
+        if self.link_page:
+            return self.link_page.url
+        return None
 
     def __str__(self):
         return self.title
 
     panels = [
+        FieldPanel('title_de'),
+        FieldPanel('title_en'),
         PageChooserPanel('link_page'),
-        FieldPanel('title')
+        StreamFieldPanel('subpages')
     ]
 
 
+class NavigationMenuItem(Orderable, MenuItem):
+    parent = ParentalKey('apps_snippets.NavigationMenu',
+                         related_name='menu_items')
+
+
```

Comment:
Since I'm getting AttributeError at /admin/pages/add/apps_home/simplepage/3/
'NoneType' object has no attribute '_inc_path' I think that this may not be working as it should. But I don't know that this is helpful information

Reply:
@sabinammm That error is solved here: https://github.com/liqd/digitalstrategie/pull/37 (hopefully!).